src/Api.js

```diff
@@ -0,0 +1,18 @@
+import axios from "axios";
+
+export const getAllArticles =()=>{
+   return axios
+    .get("https://stephennc-news.onrender.com/api/articles")
+    .then(({ data }) => {
+    return data.articles;
+    //setIsLoading(false)
+    });
+}
+
+export const getSingleArticle = (article_id) => {
+return axios
+.get(`https://stephennc-news.onrender.com/api/articles/${article_id}`)
+.then(({ data }) => {
+    return data.article;
+});
```

Comment:
nice work here - have a look at axios.create() - it sets a baseURL so you don't have to keep writing out the long URL for each request you make 👌🏻